server/src/v2/helpers/cipher.js

```diff
@@ -7,14 +7,15 @@
 */
 
 import crypto from 'crypto-js'
+import cryptoDefault from 'crypto'
```

Comment:
```suggestion
import { publicEncrypt, publicDecrypt } from 'crypto'
```
Then drop `cryptoDefault.` when used (only if those are the only 2 functions we end up using). Didn't plan well for this conflict

Reply:
The decrypt will be completed on client side, will just be using publicEncrypt as per e827b10